src/app/constants/labels.js

```diff
@@ -0,0 +1,32 @@
+export const definitionLabels = {
```

Comment:
We may want to change this to a more specific variable name and import it with a generic name to the DefinitionList component. We also may want to make that a somewhat more generic component itself and have an ItemDefinitionList extend that instead.

Reply:
Not sure what is your idea here, can you elaborate?

Reply:
Just thinking that the variable name definitionLabels for the detail's constant has a generic name and it would benefit from having a specific name like the allWorksLabels. Perhaps detailDefinitionLabels or detailLabels instead. Was just brainstorming about the component. Maybe we can chat about this today so that I have a better understanding of how components are extended in React. I'm more used to the OOP model of abstraction and extension which may not fit here.

Reply:
I changed the name of the labels